### ISSUE TITLE
Import Iterable from collections.abc instead of collections

### ISSUE THREAD

**Issue by huddlej** [neherlab/treetime#111]
Testing out treetime with Python 3.8, I got the following warning:

```
  /opt/conda/envs/py38/lib/python3.8/site-packages/treetime/distribution.py:4: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
    from collections import Iterable
```

I'm just leaving a note here so we don't forget that `from collections import Iterable` needs to be `from collections.abc import Iterable` before Python 3.9 can be supported.

**Comment by Echelon9** [comment]
Looks like a few people have noticed this one :) @tacaswell @jasongrout

Appears to be resolved by one of @tacaswell commits in PR #108 (https://github.com/neherlab/treetime/pull/108/commits/db4f9b37ff740a778b92d7305f98f950344a1ee0)

**Comment by rneher** [comment]
should be fixed in #108
